pkg/config/libpodConfig.go

```diff
@@ -197,6 +197,10 @@ func newLibpodConfig(c *Config) error {
 		return errors.Wrapf(err, "error finding config on system")
 	}
 
+	if len(configs) == 0 {
+		return nil
```

Comment:
could we put at least a log debug statement here?

Reply:
This is the default and would flood the log, so I think this is a bad idea.